src/tools/actors/ActorExecutionTrace.java

```diff
@@ -275,7 +279,17 @@ public static void mailboxExecuted(final Mailbox m, final Actor actor) {
       b.put(Events.Mailbox.id);
       b.putLong(m.getBasemessageId()); //base id for messages
       b.putLong(actor.getActorId()); //receiver of the messages
+
       int idx = 0;
+      byte eventid = MESSAGE_BASE;
+
+      if (VmSettings.MESSAGE_TIMESTAMPS) {
```

Comment:
ok, you have to explain to me why this bit is set depending on the static configuration of the interpreter. This is not depending on a dynamic value. So, why is this info relevant in each event id?

(same comment below)